beefy-gadget/src/gossip.rs

```diff
@@ -149,7 +149,7 @@ where
 				return ValidationResult::ProcessAndKeep(self.topic);
 			} else {
 				// TODO: report peer
-				debug!(target: "beefy", "🥩 Bad signature on message: {:?}, from: {:?}", msg, sender);
+				warn!(target: "beefy", "🥩 Bad signature on message: {:?}, from: {:?}", msg, sender);
```

Comment:
This is not something that depends on local user configuration, we have a convention to not trigger `warn`s for things that are not addressable by the user.